.github/workflows/ci.yml

```diff
@@ -191,6 +191,14 @@ jobs:
           gpg_private_key: ${{ secrets.CALYPTIA_GPG_KEY }}
           passphrase: ${{ secrets.CALYPTIA_GPG_KEY_PASSPHRASE }}
 
+      - name: Get the manifest
+        run: |
+          version=$(curl -s https://api.github.com/repos/calyptia/core-operator-releases/releases | jq .[0].tag_name)
+          gh release download -p manifest.yaml -R calyptia/core-operator $version
+          mv manifest.yaml cmd/operator
```

Comment:
@patrick-stephens is this the best way to get manifest.yaml and move it to cmd/operator. Embed package, for whatever reason by design, only accepts full path and can't use relative path to embed files.

Reply:
Depends, we could also checkout the repo or do something like that.

What's the goal though, which version of the manifest do you want? Is it always latest release or a specific release?

Reply:
Probably want a `curl -sSfL` just to fail explicitly and handle redirects (shouldn't be any).